core/ir/ir.cpp

```diff
@@ -52,13 +101,16 @@ c10::optional<at::ScalarType> get_value_first_calc_dtype_opt(torch::jit::Block*
   auto b_ins = b->inputs();
   std::unordered_set<torch::jit::Value*> b_in_set(b_ins.begin(), b_ins.end());
 
-  TORCHTRT_ASSERT(
-      in->type() == c10::TensorType::get(), "Input is not a tensor, cannot check for dtype based on calculation");
+  // TORCHTRT_ASSERT(
+  //     in->type() == c10::TensorType::get(), "Input is not a tensor, cannot check for dtype based on calculation");
 
   auto consumers = in->uses();
   auto search_list = std::vector<torch::jit::Use>(consumers.begin(), consumers.end());
-
-  for (auto& u : search_list) {
+  LOG_DEBUG("Users number for " << in->debugName() << ": " << consumers.size());
```

Comment:
Can we delete this. In general if this is not information that helps a user figure out what is happening, its not worth printing